iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergTableUtil.java

```diff
@@ -242,4 +242,17 @@ public static void setCurrentSnapshot(Table table, Long value) {
     manageSnapshots.setCurrentSnapshot(value);
     manageSnapshots.commit();
   }
+
+  /**
+   * Fast forwards a branch to another.
+   * @param table the iceberg table
+   * @param sourceBranch the source branch
+   * @param targetBranch the target branch
+   */
+  public static void fastForwardBranch(Table table, String sourceBranch, String targetBranch) {
+    ManageSnapshots manageSnapshots = table.manageSnapshots();
+    LOG.debug("Fast Forwarding the iceberg table {} branch {} to {}", table.name(), sourceBranch, targetBranch);
+    manageSnapshots.fastForwardBranch(sourceBranch, targetBranch);
```

Comment:
is it going to check for conflicts with the targetBranch? scenario: someone wrote to the target branch after it was branched out.

Reply:
yes, it will throw exception, I will add a -ve test for this case